requirements.txt

```diff
@@ -1,12 +1 @@
-numpy>=1.17.0
-pandas>=0.25.0
-gym>=0.15.7
-pyyaml>=5.1.2
-stochastic==0.6.0
-tensorflow>=2.7.0
-ipython>=7.12.0
-matplotlib>=3.1.1
-plotly>=4.5.0
-ipywidgets>=7.0.0  # Required for rendering
-torch>=1.10.2
-deprecated>=1.2.13
+.
```

Comment:
Will pip install -r requirements.txt still work as mentioned in the README? If not, it would be good to also update the documentation.

Reply:
It does work with a single dot in the file (which refers to the setup.py install_requires) and no behavior is changed this way. The Docker install and the TravisCI checks both use pip install -r requirements.txt the same way they did before. This prevents potential transcription errors between the install_requires of setup.py and requirements.txt. It also works with the example requirements. I thought it was better practice to not duplicate install package definitions in multiple files, again, to mitigate potential future errors and not have to update and check 3 files everytime for consistency.